Scrapping_part_3/scrapping_part3/scrapping_part3/settings.py

```diff
@@ -0,0 +1,104 @@
+# Scrapy settings for scrapping_a_website project
+#
+# For simplicity, this file contains only settings considered important or
+# commonly used. You can find more settings consulting the documentation:
+#
+#     https://docs.scrapy.org/en/latest/topics/settings.html
+#     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
+#     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
+
+
+BOT_NAME = 'scrapping_part3'
+
+SPIDER_MODULES = ['scrapping_part3.spiders']
+NEWSPIDER_MODULE = 'scrapping_part3.spiders'
+
+
+# Crawl responsibly by identifying yourself (and your website) on the user-agent
+#USER_AGENT = 'scrapping_a_website (+http://www.yourdomain.com)'
+
+# Obey robots.txt rules
+ROBOTSTXT_OBEY = False
+
+DOWNLOADER_MIDDLEWARES = {
+    'scrapping_part3.middlewares.ProxyMiddleware': 543,
+}
+REQUESTS_PER_SAME_TOR_IDENTITY = 100
+
+from rotating_proxies.policy import BanDetectionPolicy
+'''
+for using tor and docker machine
+DOWNLOADER_MIDDLEWARES = {
+    'rotating_proxies.middlewares.RotatingProxyMiddleware': 610,
+    'rotating_proxies.middlewares.BanDetectionMiddleware': 620,
+}
+
+ROTATING_PROXY_LIST_PATH = 'proxy-list.txt'
+ROTATING_PROXY_PAGE_RETRY_TIMES = 5
+'''
```

Comment:
You have to use `#` instead of triple quote syntax `'''` if you want comment something out. Btw do you still need this?